tests/parm/model_configure_hafs_shared.IN

```diff
@@ -25,19 +25,126 @@ kchunk3d:                -1
 ideflate:                @[IDEFLATE]
 nbits:                   @[NBITS]
 
-app_domain:              'regional'
-output_grid:             @[OUTPUT_GRID]
-cen_lon:                 @[CEN_LON]             # central longitude
-cen_lat:                 @[CEN_LAT]             # central latitude
-lon1:                    @[LON1]                # longitude of lower-left
-lat1:                    @[LAT1]                # latitude of lower-left
-lon2:                    @[LON2]                # longitude of upper-right
-lat2:                    @[LAT2]                # latitude of upper-right
-dlon:                    @[DLON]
-dlat:                    @[DLAT]
+output_grid:             @[OUTPUT_GRID]     # gaussian_grid, global_latlon, regional_latlon, rotated_latlon, lambert_conformal
```

Comment:
@DusanJovic-NOAA, could you please help change this line by adding the **cubed_sphere_grid** in the comment, e.g.,
```
output_grid:             @[OUTPUT_GRID]     # gaussian_grid, global_latlon, regional_latlon, rotated_latlon, lambert_conformal, cubed_sphere_grid
```
Thanks!